modules/hcp-support-matrix.adoc

```diff
@@ -0,0 +1,140 @@
+
+[id="hcp-matrix-mgmt_{context}"]
+== Management cluster support
+
+Any supported standalone {product-title} cluster can be a management cluster. The following table maps {mce-short} versions to the management cluster versions that support them:
+
+.Supported {mce-short} versions for {product-title} management clusters
+[cols="2",options="header"]
+|===
+|Management cluster version |Supported {mce-short} version
+
+|4.12 - 4.15
+|2.4
+
+|4.13 - 4.16
+|2.5
+
+|4.14 - 4.17
+|2.6
+
+|4.15 - 4.17
+|2.7
+|===
+
+[id="hcp-matrix-hc_{context}"]
+== Hosted cluster support
+
+For hosted clusters, no direct relationship exists the management cluster version and the hosted cluster version. The hosted cluster version depends on the HyperShift Operator that is included with your {mce-short} version. The following table maps {mce-short} versions to which hosted cluster versions you can create by using the HyperShift Operator that is associated with that version of {mce-short}:
+
+.Hosted cluster versions that can be created by {mce-short} versions
+[cols="5",options="header"]
+|===
+|Hosted cluster version  |{mce-short} 2.4 |{mce-short} 2.5 |{mce-short} 2.6 |{mce-short} 2.7
+
+|4.14
+|Yes
+|Yes
+|Yes
+|Yes
+
+|4.15
+|No
+|Yes
+|Yes
+|Yes
+
+|4.16
+|No
+|No
+|Yes
+|Yes
+
+|4.17
+|No
+|No
+|No
+|Yes
+|===
+
+[id="hcp-matrix-updates_{context}"]
+== Updates of {mce-short}
+
+When you update to another version of the {mce-short}, your hosted cluster can continue to run if the HyperShift Operator that is included in the version of {mce-short} supports the hosted cluster version. The following table shows the affects of {mce-short} updates on various versions of hosted clusters:
+
+.Affects of {mce-short} updates on hosted clusters
+[cols="4",options="header"]
+|===
+|Hosted cluster version| Update {mce-short} 2.4 to 2.5 |Update {mce-short} 2.5 to 2.6 |Update {mce-short} 2.6 to 2.7
+
+|{product-title} 4.14
+|No affect, hosted cluster continues to run
+|No affect, hosted cluster continues to run
+|No affect, hosted cluster continues to run
+
+|{product-title} 4.15
+|
+|No affect, hosted cluster continues to run
+|No affect, hosted cluster continues to run
+
+|{product-title} 4.16
+|
+|
+|No affect, hosted cluster continues to run
+|===
+
+For example, {mce-short} works with both {product-title} 4.14 and 4.15 clusters. If you have a {product-title} 4.14 hosted cluster on the management cluster and you update from {mce-short} 2.4 to 2.5, the hosted cluster can continue to run.
+
+[id="hcp-matrix-platform_{context}"]
+== Platform support
+
+The following table indicates which {product-title} versions are supported for each platform of {hcp}. In the table, management cluster version refers to the {product-title} version where the {mce-short} is enabled:
+
+.Required {product-title} versions for platforms
+[cols="3",options="header"]
+|===
+|Platform |Management cluster version |Hosted cluster version
```

Comment:
I was thinking to decouple the platform table. We would like to move to a model the management cluster can be any supported platform, and the hosted cluster platform one of the supported HCP platforms. 

The exception I can think of is for AWS: if the management cluster is on-prem, there is no way to do private-link or create private clusters.